src/swh_api_functions.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+import requests
+
+# class to genrate our own custom response
+from custom_api_response import CustomApiResponse
+
+BASE_SWH_API_URL = "https://archive.softwareheritage.org/api/1/"
+
+
+def get_content_information(hash_type, hash):
+
+    """
+    this function returns information like checksums, data_url, file_url, license_url
```

Comment:
Check the style of comments used in scancode-toolkit code
We like imperative style for docstrings. This is eventually doc that's designed to end up in generated HTML web pages on readthedocs